extensions/amp-ad/0.1/test/test-amp-ad-3p-impl.js

```diff
@@ -212,6 +212,23 @@ function tests(name) {
       })).to.be.not.be.rejected;
     });
 
+    it('should only layout once', () => {
+      return getAd({
+        width: 300,
+        height: 250,
+        type: '_ping_',
+        src: 'testsrc',
+      }, 'https://schema.org').then(ad => {
+        const firstLayout = ad.implementation_.layoutCallback();
```

Comment:
we can, but then we will be testing `layoutCallback()` in custom-element, instead of the overridden BaseElement`layoutCallback()` function for `amp-ad`. Is that what we want to test?